common.js

```diff
@@ -5722,6 +5722,122 @@ const allSoloCars = {
       'all': ['hs', 'fsp', 'dp'],
     },
   },
+  'Volvo': {
+    'S60 & V60 Polestar': {
+      '2016': ['ds'],
+      '2017': ['ds'],
+      '2018': ['ds'],
+      '2019': ['ds'],
+      '2020': ['ds']
+    },
+    'S60R (except Polestar)': {
+      'all': ['gs']
+    },
+    'V70R (except Polestar)': {
+      'all': ['gs']
+    },
+    'Turbo models (NOC)': {
+      'all': ['gs', 'gt']
+    },
+    'C30': {
+      '2006': ['hs', 'dsp'],
+      '2007': ['hs', 'dsp'],
+      '2008': ['hs', 'dsp'],
+      '2009': ['hs', 'dsp']
+    },
+    'NOC': {
+      'all': ['hs']
+    },
+    'S60R': {
```

Comment:
Combine this with the previous S60R, valid classes are `gs`, `stu`, `asp`. Remove the other duplicate models.

Reply:
Corrected!